src/avram/query_builder.cr

```diff
@@ -226,9 +216,7 @@ class Avram::QueryBuilder
   end
 
   def ordered?
-    @orders.values.any? do |columns|
-      columns.any?
-    end
+    @orders.size > 0
```

Comment:
I *think* you can do `@orders.any?`. But if not this is good!

Reply:
they both work. I just ran a small benchmark, it turns out if `@orders` is actually empty, then `@orders.size` is faster. If there's stuff in it, then `@orders.any` is faster. However, the speed difference is negligible, so I think I'll roll with any